include/sway/config.h

```diff
@@ -189,6 +189,8 @@ struct input_config {
 
 	bool capturable;
 	struct wlr_box region;
+
+	int seq_id;
```

Comment:
Just to make it extra safe to not overflow can we make these `uint64_t`?

Reply:
Now `uint64_t`, just in case you managed to work your way through 4294967296 input configs... :)